_site/open-source-day-2021-summer-mentors/index.html

```diff
@@ -267,6 +267,1196 @@ <h1 id="page-title" class="page__title" itemprop="headline">Your Mentors
       <section class="page__content" itemprop="text">
         
         <p>OSD Mentors work with one of our partner projects to help participants contribute to their open source projects. Mentors are selected primarily based on project needs and should plan to attend the entirety of OSD.</p>
+  <style type="text/css">
+    .mentor-details {
+      background: linear-gradient(59deg, #4892c7 0%, #b3ccdd 64%);
+      border-radius: 10px;
+      padding: 10px 80px;
+    }
+  </style>
+<ul>
+
+  <li>
+    <h3>Augur</h3>
+  
+    <ul>
+      
+
+
+<div itemscope="" itemtype="https://schema.org/Person" class="mentor-details">
```

Comment:
`class="mentor-details"` needs to be added to` _includes/mentor-profile.html` instead of here.